staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -4265,6 +4265,7 @@ type ServicePort struct {
 
 	// The IP protocol for this port. Supports "TCP", "UDP", and "SCTP".
 	// Default is TCP.
+	// +default="TCP"
```

Comment:
@liggitt Should we back-port that in 1.20? If so, we should maybe make that into a separate PR.

Reply:
It's in a separate commit we can pick if needed. If it's not a regression in 1.20 and we don't have calls for it in bugs, I'd probably just let it ride in 1.21